app/src/main/kotlin/com/passwordstore/android/ui/activity/BaseActivity.kt

```diff
@@ -12,13 +12,15 @@ import com.passwordstore.android.utils.Authenticator
 
 abstract class BaseActivity : AppCompatActivity() {
 
+    private val passwordStoreApplication by lazy { application as PasswordStoreApplication }
```

Comment:
This variable name is far too long. Abbreviate it to just `app` or `application` depending on your thoughts about name shadowing.

Reply:
Fixing